code/rd-vanilla/G2_API.cpp

```diff
@@ -1401,6 +1413,15 @@ qboolean G2API_StopBoneAngles(CGhoul2Info *ghlInfo, const char *boneName)
 	return ret;
 }
 
+#ifdef JK2_MODE
```

Comment:
What's the benefit of not compiling all of these in JK2? They're not used by JK2 so they won't ever be run.